js/src/Ice/InstanceExtensions.js

```diff
@@ -181,6 +182,12 @@ Instance.prototype.finishSetup = function (communicator) {
             this._initData.properties.getIcePropertyAsInt("Ice.Connection.Client.InactivityTimeout"),
         );
 
+        if (FileLogger !== null && this._initData.logger === null) {
+            const logfile = this._initData.properties.getIceProperty("Ice.LogFile");
+            if (logfile.length > 0) {
+                this._initData.logger = new FileLogger("", logfile);
```

Comment:
Why is the prefix always empty?

Reply:
I guess we can use Ice.ProgramName as the default?

We have to fix process logger too:

https://github.com/zeroc-ice/ice/blob/0cca8df118385badf630932c306fc35e59814f85/js/src/Ice/ProcessLogger.js#L12